scripts/build_industrial_energy_demand_per_country_today.py

```diff
@@ -227,6 +227,18 @@ def industrial_energy_demand(countries, year):
     return pd.concat(demand_l, keys=countries)
 
 
+def add_coke_ovens(demand, fn, year):
+    df = pd.read_csv(fn, index_col=[0,1]).xs(year, level=1)
+    df = df.rename(columns={'Total all products':'Total'})[fuels.keys()]
+    df = df.rename(columns=fuels).T.groupby(level=0).sum().T
+    df["other"] = df["all"] - df.loc[:,df.columns != "all"].sum(axis=1)
+    df = df.T.reindex_like(demand.xs("Integrated steelworks", axis=1, level=1)).fillna(0)
+    sel = demand.columns.get_level_values(1)=="Integrated steelworks"
+    demand.loc[:,sel] += 0.75 * df.values
```

Comment:
What is the 0.75? Perhaps document the unit?

Reply:
The unit and meaning of the factor was documented in this [commit](https://github.com/PyPSA/pypsa-eur/pull/1167/commits/9b8139abbea7cdfd06625575b392ab71d382a9d1)